src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "notenoughupdates",
+  "version": "9999.0",
+  "schemaVersion": 1,
+  "entrypoints": {
+    "main": [
+      "io.github.moulberry.notenoughupdates.envcheck.FabricEntrypoint"
```

Comment:
This can be replaced with `io.github.moulberry.notenoughupdates.envcheck.EnvironmentScan::checkForgeEnvironment` which would remove the need of having an entry point class, it may require that method to be public instead of package private.

Reply:
I am aware, but I just dont think thats a concern for me right now